composer.json

```diff
@@ -25,13 +25,15 @@
         "php": "^8.1",
         "ext-mongodb": "^1.15",
         "composer-runtime-api": "^2.0.0",
-        "illuminate/support": "^10.0|^11",
+        "illuminate/cache": "^10.36|^11",
```

Comment:
Required to access `Cache::isOwnedCurrentProcess`
https://github.com/laravel/framework/commit/234444e8fb17daf16a703877e3fb8ccb4b84cad4